packages/client/src/intros/shared/Intro.tsx

```diff
@@ -43,14 +45,18 @@ export default ({
 
   return (
     <>
-      <Paragraph gutterBottom={usableForBullets.length > 0 ? 8 : undefined}>
-        {t("introPage.common.check for permit intro")}{" "}
-        {usableForBullets.length > 0 && t("introPage.common.permit for")}
+      { needReport && <Paragraph>
+        {t("introPage.demolition.need report")}{" "}
+      </Paragraph> }
+
+      <Paragraph gutterBottom={usableForBullets?.length > 0 ? 8 : undefined}>
+        {!needReport && t("introPage.common.check for permit intro")}{" "}
+        {usableForBullets?.length > 0 && t("introPage.common.permit for")}
```

Comment:
This is wrong, because now this `<Paragraph>` can be rendered with empty content. Can you not do:
`{needReport ? {t("introPage.demolition.need report")} ? t("introPage.common.check for permit intro")}}` ?
If this is not possible, then this paragraph needs conditional rendering as well